internal/api/v1alpha1/authproxyworkload_types.go

```diff
@@ -239,6 +239,16 @@ type InstanceSpec struct {
 	// Optional, when set this environment variable will be added to all containers in the workload.
 	//+kubebuilder:validation:Optional
 	HostEnvName string `json:"hostEnvName,omitempty"`
+
+	// UnixSocketPath is the path to the unix socket where the proxy will listen
+	// for connnections. This will be mounted to all containers in the pod.
+	//+kubebuilder:validation:Optional
+	UnixSocketPath string `json:"unixSocketPath,omitempty"`
```

Comment:
Why do we have this field and the one below?

Reply:
What you said makes a lot of sense. Can we work that into the documentation here?